stixcore/processing/L0toL1.py

```diff
@@ -1,38 +1,53 @@
+import logging
 from time import perf_counter
 from pathlib import Path
 
+from sunpy.util.datatype_factory_base import NoMatchError
+
 from stixcore.io.fits.processors import FitsL1Processor
-from stixcore.products.product import Product
+from stixcore.products import Product
 from stixcore.util.logging import get_logger
 
-logger = get_logger(__name__)
+logger = get_logger(__name__, level=logging.DEBUG)
 
 
 class Level1:
     def __init__(self, source_dir, output_dir):
         self.source_dir = Path(source_dir)
         self.output_dir = Path(output_dir)
-        self.level0_files = sorted(list(self.source_dir.rglob('*.fits')))
+        self.level0_files = sorted(list(self.source_dir.rglob('*.fits')))[-10:]
         self.processor = FitsL1Processor(self.output_dir)
```

Comment:
is this only for testing?
does this mean we always process just the 10 recent ones?

Reply:
Yea just a leftover from some testing - I mean I don't know of these script will the real one we use just what I have at the moment.